src/applications/disability-benefits/2346/components/supplyCustomFields/deviceNameField.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+const deviceNameField = () => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Why does this break the [rules of hooks](https://reactjs.org/docs/hooks-rules.html)?  do we need to update the eslint plugin?

Reply:
Yeah we need to update eslint.

Reply:
Isn't ESLint complaining that this is being called in a function rather than at the top level of the component? https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
Correct I removed these custom fields because they are no longer needed.

Reply:
Sorry I misread the code because the name of the functional component didn't start with an uppercase letter. So I thought this call to `useState` was in the wrong place. In this case if you'd changed the name of functional component to `DeviceFieldName` to stay in line with best practices, ESLine wouldn't complain either.

I realize you've since removed this `useState` call, but you should change the names of all your functional components (and the file names themselves) to have a leading capital letter. I see a few instances of that in this PR.